packages/client/test/FakeBrubeckNodeRegistry.ts

```diff
@@ -0,0 +1,30 @@
+import { EthereumAddress } from 'streamr-client-protocol'
+import { Lifecycle, scoped } from 'tsyringe'
+import { FakeBrubeckNode } from './FakeBrubeckNode'
+
+@scoped(Lifecycle.ContainerScoped)
+export class FakeBrubeckNodeRegistry {
```

Comment:
This doesn't correspond to any existing class does it? If not I find the naming confusing, firstly the `Fake` part (there is no real that that is replaced?) and `Registry` part (this isn't a smart contract thing?)

Was this the singleton thing? Maybe something like `ActiveNodesSingleton` etc.?

Reply:
Renamed